onnx/common/constants.h

```diff
@@ -13,7 +13,18 @@ namespace ONNX_NAMESPACE {
 constexpr const char* AI_ONNX_ML_DOMAIN = "ai.onnx.ml";
 constexpr const char* AI_ONNX_TRAINING_DOMAIN = "ai.onnx.training";
 constexpr const char* AI_ONNX_PREVIEW_TRAINING_DOMAIN = "ai.onnx.preview.training";
+// The following two are equivalent in an onnx proto representation.
 constexpr const char* ONNX_DOMAIN = "";
+constexpr const char* AI_ONNX_DOMAIN = "ai.onnx";
+
+inline std::string NormalizeDomain(std::string domain) {
```

Comment:
Why std::string here and const std::string& just below?

Reply:
Fixed, thanks!